docs/reference/auditing.md

```diff
@@ -16,38 +16,240 @@ lakeFS Cloud
 {: .note}
 > Auditing is only available for [lakeFS Cloud]({% link understand/lakefs-cloud.md %}).
 
-The lakeFS audit log allows you to view all relevant user action information in a clear and organized table, including when the action was performed, by whom, and what it was they did. 
+{: .warning }
+> Please note, as of Jan 2024, the queryable interface within the lakeFS Cloud UI has been removed in favor of direct access to lakeFS audit logs. This document now describes how to set up and query this information using [AWS Glue](https://aws.amazon.com/glue/) as a reference.
 
-![audit log]({% link assets/img/audit-log.png %})
+The lakeFS audit log allows you to view all relevant user action information in a clear and organized table, including when the action was performed, by whom, and what it was they did. 
 
 This can be useful for several purposes, including: 
 
 1. Compliance - Audit logs can be used to show what data users accessed, as well as any changes they made to user management.
 
 2. Troubleshooting - If something changes on your underlying object store that you weren't expecting, such as a big file suddenly breaking into thousands of smaller files, you can use the audit log to find out what action led to this change. 
 
-## Audit Fields
+## Setting up access to Audit Logs on AWS S3
+
+The access to the Audit Logs is done via [AWS S3 Access Point](https://aws.amazon.com/s3/features/access-points/).
+
+There are different ways to interact with an access point (see [Using access points in AWS](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-access-points.html)).
+
+The initial setup:
+
+1. Take note of the IAM Role ARN that will be used to access the data. This should be the user or role used by e.g. Athena.
+1. [Reach out to customer success](mailto:support@treeverse.io?subject=ARN to use for audit logs) and provide this ARN. Once receiving the ARN role, an access point will be created and you should get in response the following details:
+   1. S3 Bucket (e.g. `arn:aws:s3:::lakefs-audit-logs-us-east-1-production`)
+   2. S3 URI to an access point (e.g. `s3://arn:aws:s3:us-east-1:924819537486:accesspoint/lakefs-logs-<organization>`)
```

Comment:
I always reluctant when it comes to publishing AWS Account ID, can we change that everywhere to a place holder? 
i.e `arn:aws:s3:us-east-1:<treeverse-account>:accesspoint`